api/src/main/java/com/gentics/mesh/etc/config/ClusterOptions.java

```diff
@@ -93,6 +95,11 @@ public class ClusterOptions implements Option {
 	@EnvironmentVariable(name = MESH_CLUSTER_TOPOLOGY_CHANGE_READONLY_ENV, description = "Override the topology change readonly flag.")
 	private boolean topologyChangeReadOnly = DEFAULT_TOPOLOGY_CHANGE_READONLY;
 
+	@JsonProperty(required = false)
+	@JsonPropertyDescription("The path of the hazelcast configuration path")
+	@EnvironmentVariable(name = MESH_CLUSTER_HAZELCAST_CONFIG_PATH_ENV, description = "Override the default hazelcast configuration path")
+	private String hazelcastConfigPath = DEFAULT_HAZELCAST_CONFIG_PATH;
```

Comment:
Please add a changelog entry for this change.

Reply:
Now that I think about it I think this configuration option should be moved to mdm since it's not used by the orientdb version

Reply:
Do we need this configurable anyways? I suppose the configuration path is set by OrientDB, if OrientDB is creating the Hazelcast instance. So maybe we should just set it to the same value in MDM?

Reply:
Yes, we could check the system property hazelcast.config, if not set, we hard code it to config/hazelcast.xml.
This way we allow user to override it if necessary

Reply:
I removed the configuration option